src/main/java/com/suse/saltstack/netapi/calls/modules/Match.java

```diff
@@ -0,0 +1,61 @@
+package com.suse.saltstack.netapi.calls.modules;
+
+import com.google.gson.reflect.TypeToken;
+import com.suse.saltstack.netapi.calls.LocalCall;
+import com.suse.saltstack.netapi.utils.ClientUtils;
+
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * salt.modules.match
+ */
+public class Match {
+
+    public static LocalCall<Boolean> compound(String tgt, Optional<String> minionId) {
```

Comment:
It makes sense to have a simple javadoc, what will popup in the IDE, so everyone can see it while programming. I don't know what means "tgt". Inability to write "target"? That Goofy Thing? Tyrese, Ginuwine and Tank R&B singers?..